### PR TITLE
Update demos, use some better screenshots from demozoo

### PR DESCRIPTION
**Pull request by laurentlb**

### CODE REVIEW

File: demos.js [hunk 1]
```diff
@@ -86,7 +86,7 @@ const allDemos = [
         "group": "Conspiracy",
         "youtube": "LJaYJQB2tc0",
         "date": "2004-08-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00013/00013030.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/90/74/cfee.14495.png",
```
Comment (virtualzavie) [inline demos.js]: What about:
https://media.demozoo.org/screens/o/02/69/aa7d.14468.png

File: demos.js [hunk 2]
```diff
@@ -95,7 +95,7 @@ const allDemos = [
         "group": "Hoodlum and Tristar & Red Sector Inc. and Paradox",
         "youtube": "tTN81zaiP3U",
         "date": "2004-11-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00014/00014107.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/31/2c/66b6.154362.png",
```
Comment (virtualzavie) [inline demos.js]: I'd suggest removing this entry entirely. It's unlikely to be interesting to someone looking to discover 64k intros.

File: demos.js [hunk 3]
```diff
@@ -266,7 +266,7 @@ const allDemos = [
         "group": "ate bit",
         "youtube": "jhXHZ9ZfsKI",
         "date": "2009-08-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00053/00053646.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/da/77/7948.322231.jpg",
```
Comment (virtualzavie) [inline demos.js:260]: I think the available screenshots don't do justice to this prod. We might want to take make one ourselves at some point.

File: demos.js [hunk 4]
```diff
@@ -392,18 +392,9 @@ const allDemos = [
         "group": "Razor 1911",
         "youtube": "ji2NCE_yPLQ",
         "date": "2011-04-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00056/00056871.png",
+        "screenshot": "https://media.demozoo.org/screens/o/5d/e0/c339.138822.png",
```
Comment (virtualzavie) [inline demos.js:386]: What about:
https://media.demozoo.org/screens/o/04/6b/2e59.138821.png

File: demos.js [hunk 5]
```diff
@@ -446,7 +437,7 @@ const allDemos = [
         "group": "Mercury",
         "youtube": "I8pRJ0q8OlY",
         "date": "2011-12-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00058/00058262.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/77/02/1aec.7231.jpg",
```
Comment (virtualzavie) [inline demos.js]: I would suggest either of:
https://media.demozoo.org/screens/o/58/dc/c546.7236.jpg
https://media.demozoo.org/screens/o/05/07/4df5.7241.jpg
https://media.demozoo.org/screens/o/5b/66/77da.7238.jpg

File: demos.js [hunk 6]
```diff
@@ -734,7 +716,7 @@ const allDemos = [
         "group": "Logicoma",
         "youtube": "GjuridCR2Fo",
         "date": "2017-04-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00069/00069658.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/53/91/cb3c.132627.jpg",
```
Comment (virtualzavie) [inline demos.js]: What about:
https://media.demozoo.org/screens/o/42/99/a8e5.132639.jpg

File: demos.js [hunk 7]
```diff
@@ -761,7 +743,7 @@ const allDemos = [
         "group": "Poo-Brain",
         "youtube": "EIp7ffUrUCs",
         "date": "2017-08-15",
-        "screenshot": "https://content.pouet.net/files/screenshots/00071/00071570.jpg",
+        "screenshot": "https://media.demozoo.org/screens/o/d1/7a/012a.146494.jpg",
```
Comment (virtualzavie) [inline demos.js]: The other screenshot of the same scene could make a more interesting preview:
https://media.demozoo.org/screens/o/22/14/5827.146493.jpg

**Review by virtualzavie — APPROVED** [review]
Nice.
I made a few suggestions as well.

**Review by laurentlb — COMMENTED** [review]
lgtm

**Pull request merged**